test/integration/image-optimizer/test/index.test.js

```diff
@@ -55,12 +55,6 @@ function runTests({ w, isDev, domains = [], ttl, isSharp }) {
     expect(await res.text()).toMatch(/Image Optimizer Home/m)
   })
 
-  it('should handle non-ascii characters in image url', async () => {
-    const query = { w, q: 90, url: '/äöü.png' }
-    const res = await fetchViaHTTP(appPort, '/_next/image', query, {})
-    expect(res.status).toBe(200)
-  })
```

Comment:
This was testing the API without testing how the component uses the API which is totally irrelevant